docs/vtpm.md

```diff
@@ -0,0 +1,122 @@
+# vTPM
```

Comment:
I would really try to avoid the `v` in front of TPM given it's implicitly always virtual. The VMM emulates a TPM device, which is why it can be seen as virtual.